<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageSensitivity</class>
 <widget class="QWidget" name="PageSensitivity">
  <layout class="QFormLayout" name="layout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelX">
     <property name="buddy">
      <cstring>sliderX</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSlider" name="sliderX">
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelY">
     <property name="buddy">
      <cstring>sliderY</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="sliderY">
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sliderX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PageSensitivity</receiver>
   <slot>onSensitivityXChanged(int)</slot>
  </connection>
  <connection>
   <sender>sliderY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PageSensitivity</receiver>
   <slot>onSensitivityYChanged(int)</slot>
  </connection>
 </connections>
</ui>
